test/config/basic/bin/bundle.js

```diff
@@ -0,0 +1,100 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
Should not be committed

Reply:
okay I will remove it !

Reply:
@evenstensberg should I add `test/**/bin/` in `.gitignore` ?

Reply:
Add a `.gitignore` inside your `config` folder. It's better, so we don't include files by accident.

Reply:
done